app/src/app/extensions.module.ts

```diff
@@ -29,12 +29,13 @@ import { AcaAboutModule } from '@alfresco/aca-about';
 import { AcaSettingsModule } from '@alfresco/aca-settings';
 import { AcaFolderRulesModule } from '@alfresco/aca-folder-rules';
 import { environment } from '../environments/environment';
+import packageJson from 'package.json';
 
 @NgModule({
   imports: [
     AosExtensionModule,
```

Comment:
Not sure about this. Does it mean that content-ee will use the package of aca about?

Reply:
this module is going to be overridden  by `content-ee` with its own `package.json` file